extras/jenkins/Linode/Jenkinsfile

```diff
@@ -69,23 +69,23 @@ pipeline {
             # Create the directory for the kubeconfig
             mkdir -p $HOME/.kube || true
             chmod 777 $HOME/.kube || true
-            '''
+             '''
       }
     }
 
     stage('Cleaning Up') {
       steps {
 
         /*
-         * Run a find and check for any stacks that currently exist with our generated stack name; this should not
-         * happen in normal operation, but could potentially happen if things break so better safe than sorry.
+         * This is currently empty since we are building a new executor for each run. However, maintaining
+         * here for anyone who wants to add cleanup steps for their environment
          *
          * Other cleanup related functions can be placed here as well.
          */
 
         sh '''
-          # Clean up the Pulumi stack if it exists for our run - which it shouldn\'t, but you never know.
-          find  $WORKSPACE -mindepth 2 -maxdepth 7 -type f -name Pulumi.yaml -execdir $WORKSPACE/pulumi/python/venv/bin/pulumi stack rm marajenklke${BUILD_NUMBER}  --force --yes  \\;
+          # Just return...
```

Comment:
nit: I think this comment isn't adding anything the better comment above doesn't

Reply:
Fixed in the last push; I also made a note to go through the jenkinsfiles and do a cleanup with the idea of writing it up either for an in-repo note or as a blog post...